packages/console/src/pages/EnterpriseSsoDetails/Connection/index.tsx

```diff
@@ -42,7 +41,11 @@ function Connection<T extends SsoProviderName>({ isDeleted, data, onUpdated }: P
 
   const api = useApi({ hideErrorToast: true });
 
-  const methods = useForm<GuideFormType<T>>();
+  const methods = useForm<GuideFormType<T>>({
+    // Make typescript happy
+    // eslint-disable-next-line no-restricted-syntax
+    defaultValues: config as DeepPartial<GuideFormType<T>>,
```

Comment:
We can pick up every fields? By doing so, we can avoid some unpexected fields for the future

Reply:
I don't quite get this, looks hacky